libs/SalesforceSDK/src/com/salesforce/androidsdk/config/LoginServerManager.java

```diff
@@ -393,11 +391,11 @@ private void persistLoginServer(String name, String url, boolean isCustom, Share
 		if (numServers == 0) {
 			return null;
 		}
-		final List<LoginServer> allServers = new ArrayList<LoginServer>();
```

Comment:
if shared prefs containing urls are coming from previous versions and are malformed, will they be remain malformed when the app is updated?

Reply:
Fixed.